planet/System.ocg/Player.c

```diff
@@ -5,17 +5,29 @@
 	@author timi, Maikel, Joern, Zapper, Randrian
 */
 
-// Returns the player number of plr_name, or none if there is no such player. (written by timi for CR/CE/CP)
+static const Player = new _Player
+{
+	// _Player contains all the scripting functions for players
+	// Player is the prototype for all players and can be expanded
+	// with data or functions.
```

Comment:
So do you plan on moving some of the other functions here into the proplist?

Reply:
That was more of a documentation what the proplist is intended for. There are no major plans for the content at the moment, but in theory a rule, goal, or whatever can add its data and functions to the player proplist prototype. Say there is a goal where you have to rescue wipfs could have a function PlayScoreJingle() that plays a sound for the player who rescued a wipf (without having cluttering the global namespace).

Reply:
Okay. I guess in principle, any function in that file that takes an `int player` argument could be moved to the proplist?

Reply:
Yes. I'd leave that for another PR then? Not sure how it is intended: merge one big thing, or merge multiple PRs that are small step into the general direction? I can also do the changes separately on my fork and merge them into this PR.

Reply:
Doing that in a later PR is fine.